docs/examples/modalalert/doClearCommunicate.js

```diff
@@ -1,33 +1,48 @@
 // @flow strict
-import { type Node } from 'react';
-import { ModalAlert, CompositeZIndex, FixedZIndex, Layer, Text } from 'gestalt';
-
-const HEADER_ZINDEX = new FixedZIndex(10);
-const zIndex = new CompositeZIndex([HEADER_ZINDEX]);
+import { Fragment, useState, type Node } from 'react';
+import { Box, Button, ModalAlert, CompositeZIndex, FixedZIndex, Layer, Text } from 'gestalt';
 
 export default function DoClearCommunicate(): Node {
+  const [showComponent, setShowComponent] = useState(false);
```

Comment:
These should be initialized as `true` so the modals are shown as the user scrolls through the page, yes?